uint8 VL53L0X_GOOD_ACCURACY_MODE      = 0   # Good Accuracy mode
uint8 VL53L0X_BETTER_ACCURACY_MODE    = 1   # Better Accuracy mode
uint8 VL53L0X_BEST_ACCURACY_MODE      = 2   # Best Accuracy mode
uint8 VL53L0X_LONG_RANGE_MODE         = 3   # Longe Range mode
uint8 VL53L0X_HIGH_SPEED_MODE         = 4   # High Speed mode

# One of VL53L0X_*
uint8 mode
---
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages
